hw/formal/tools/jaspergold/parse-formal-report.py

```diff
@@ -62,7 +62,7 @@ def parse_fpv_message(str_buffer):
     return extract_messages(str_buffer, err_warn_patterns)
 
 
-def parse_fpv_summary(str_buffer):
+def parse_formal_summary(str_buffer):
```

Comment:
nit: This should probably be `get_summary`

Reply:
Updated, thanks Sri